agent/exploits/cve_2024_3273.py

```diff
@@ -0,0 +1,29 @@
+"""Agent Asteroid implementation for CVE-2024-3273"""
+
+import re
+from agent import definitions
+from agent.exploits import webexploit
+
+VULNERABILITY_TITLE = "D-Link Multiple NAS Devices Command Injection Vulnerability"
+VULNERABILITY_REFERENCE = "CVE-2024-3273"
+VULNERABILITY_DESCRIPTION = (
+    "D-Link DNS-320L, DNS-325, DNS-327L, and DNS-340L contain a command injection vulnerability. "
+    "When combined with CVE-2024-3272, this can lead to remote, unauthorized code execution."
+)
+RISK_RATING = "CRITICAL"
+
+
+class CVE20243273Exploit(webexploit.WebExploit):
```

Comment:
You need to register it!?

Reply:
Good catch! 3 other exploits had the same issue